tests/test_mapper.py

```diff
@@ -148,6 +148,7 @@ def test_get_all_values_per_clean_name(self, all_fixture1_values, mapper):
     def test_get_stats_and_get_field_result_from_stats(self, mock_get_user_choice, mock_get_user_input,
                                                        values, expected_stats, expected_field_result, mapper):
         result = mapper._get_stats(field_name='blah', items=values)
+        print(result)
```

Comment:
Do we want to keep a print in tests?

Reply:
Nope, this was me debugging, this PR isn't ready to merge, I still need to add tests to the stats interface.